components/reusable-components/projectCard/projectCard.tsx

```diff
@@ -24,6 +25,10 @@ const ProjectCard = ({
 }: ProjectCardProps) => {
   const { theme } = useTheme();
   const darkTheme = theme === 'dark';
+
+  const maxLength = 20;
```

Comment:
it is a convention to use all uppercase letters for const with immutable value. change to MAXLENGHT or ideally a more descriptive name